package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "fec-primary-player",
+  "version": "1.0.0",
+  "description": "Front End Capstone - primary player service for Sound Cloud",
+  "main": "index.js",
+  "scripts": {
+    "test": "test",
+    "serve": "nodemon server/index.js",
+    "build-sass": "node-sass -w client/src/css -o client/dist",
+    "build-react": "webpack --watch",
+    "seed": "./database/seed.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Messier-51/primary-player.git"
+  },
+  "author": "Chad Mueller",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Messier-51/primary-player/issues"
+  },
+  "homepage": "https://github.com/Messier-51/primary-player#readme",
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "express": "^4.17.1",
+    "mongoose": "^5.8.9",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "wavesurfer": "^1.3.4"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.8.3",
+    "@babel/preset-react": "^7.8.3",
+    "babel-loader": "^8.0.6",
+    "faker": "^4.1.0",
+    "node-sass": "^4.13.1",
+    "nodemon": "^2.0.2",
+    "webpack": "^4.41.5",
+    "webpack-cli": "^3.3.10"
```

Comment:
eslint missing from devDependencies (per our discussions that is what we'll use).